pdf/src/campPrint/RichText.vue

```diff
@@ -70,6 +70,23 @@ const rules = [
   {
     shouldProcessNode: (node) => node.type === 'tag' && node.name === 'li',
     processNode: (node, parent) => {
+      const childWithEmptyText = {
+        type: 'text',
+        name: '',
+        attrs: {},
+        children: [],
+        content: '',
```

Comment:
Are all of these attributes really needed? I'm not sure whether `name` is even a legal attribute, and I thought a `content` was only used in `type: 'text_instance'` nodes, but not sure...

Reply:
name is checked above, for the text node it is not checked.
So i explicitly set the name to empty.
but i also can leave it out.

Reply:
done